src/current/v25.3/ldap-authentication.md

```diff
@@ -79,35 +88,57 @@ If you also intend to configure LDAP Authorization, you will need to include an
 SET CLUSTER SETTING server.ldap_authentication.client.tls_key = '<PEM_ENCODED_KEY>';
 ~~~
 
-### Step 4: Sync database users
+### Step 4: Configure user creation
 
-{{site.data.alerts.callout_info}}
-LDAP authentication cannot be used for the `root` user or other [reserved identities]({% link {{ page.version.version }}/security-reference/authorization.md %}#reserved-identities). Credentials for `root` must be managed separately using password authentication to ensure continuous administrative access regardless of LDAP availability.
-{{site.data.alerts.end}}
+CockroachDB supports two approaches for the creation of users who will authenticate via LDAP:
 
-Before LDAP authentication can be used for a user, the username must be created directly in CockroachDB. You will need to establish an automated method for keeping users in sync with the directory server, creating and dropping them as needed.
+#### Option 1: Automatic user provisioning (recommended)
 
-For Active Directory deployments, the CockroachDB username can typically be set to match the `sAMAccountName` field from the `user` object. This field name would need to be specified in the HBA configuration using `ldapsearchattribute=sAMAccountName`.
+{% include_cached new-in.html version="v25.3" %}
+
+With automatic user provisioning, CockroachDB creates users automatically during their first successful LDAP authentication. This eliminates the need for custom scrpting to create user accounts.
+
+When enabled:
+
+- Users are created automatically upon successful LDAP authentication.
+- All auto-provisioned users receive a `PROVISIONSRC` role option set to `ldap:<ldap_server>`.
+- The `estimated_last_login_time` is tracked for auditing purposes.
+- Auto-provisioned users cannot change their own passwords (managed via LDAP only).
+
+For Active Directory deployments, the CockroachDB username will match the `sAMAccountName` field from the `user` object. This field name should be specified in the HBA configuration using `ldapsearchattribute=sAMAccountName`. Ensure that the values in the field you are using for `ldapsearchattribute` meet the CockroachDB [requirements for usernames]({% link {{ page.version.version }}/create-user.md %}#user-names).
 
 {{site.data.alerts.callout_info}}
-SQL usernames must comply with CockroachDB's [username requirements]({% link {{ page.version.version }}/create-user.md %}#user-names). Ensure that the values in the field you are using for `ldapsearchattribute` meet these requirements.
+Before you enable automatic user provisioning, it is recommended that you enable [LDAP Authorization]({% link {{ page.version.version }}/ldap-authorization.md %}). This will ensure that upon initial login, new CockroachDB users are members of the intended CockroachDB roles, with the privileges they confer, according to users' group memberships in the directory. Otherwise, functionality may be limited for a new user until your alternative process applies roles or privileges.
```

Comment:
```suggestion
Before you enable automatic user provisioning, it is recommended that you enable [LDAP Authorization]({% link {{ page.version.version }}/ldap-authorization.md %}). This ensures that upon initial login, new CockroachDB users are members of the intended CockroachDB roles, with the privileges they confer, according to users' group memberships in the directory. Otherwise, functionality may be limited for a new user until your alternative process applies roles or privileges.
```